src/game/GameVersion.cc

```diff
@@ -1,10 +1,12 @@
 #include "GameVersion.h"
 
 #ifndef GAME_VERSION
-	#define GAME_VERSION "XXXXX"
+	#define GAME_VERSION XXXXX
```

Comment:
Can this still be reached?

Reply:
Same as before, it not reached if our build system is used.
Should I remove?

Reply:
I'd remove it, yes, since it's quite sane to expect people to use our build system. :)

Reply:
Done